setup.py

```diff
@@ -12,7 +12,7 @@
     license='MIT',
     packages=find_packages(exclude=('example', 'tests')),
     install_requires=[
-        'Django>=1.8',
+        'Django>=1.7',
```

Comment:
Why did you change this to 1.7? django-two-factor-auth doesn't support 1.7. Django 1.7 doesn't get any security updates anymore either.